src/System.Private.CoreLib/shared/System/TimeZoneInfo.cs

```diff
@@ -1917,12 +1917,6 @@ private static TimeZoneInfoResult TryGetTimeZoneFromLocalMachine(string id, bool
             return result;
         }
 
-        /// <summary>
-        /// Helper function that validates the TimeSpan is within +/- 14.0 hours
```

Comment:
This is not removed - it is moved a little bit down so that similar subroutines are closer to each other